README.md

```diff
@@ -1,102 +1,38 @@
 
-## Try TypeSpec without installing anything
+## [Installation](https://typespec.io/docs)
 
-You can try TypeSpec on the web without installing anything.
-
-- [TypeSpec playground](https://microsoft.github.io/typespec/playground)
-- [TypeSpec playground for Azure services](https://azure.github.io/typespec-azure/playground)
-
-## Getting Started
-
-For documentation on the TypeSpec language, see https://microsoft.github.io/typespec.
-
-### Using Docker
-
-[See Docker documentation](./docker)
-
-### Using Node & Npm
-
-#### One-time Setup
-
-1. Install [Node.js 16 LTS](https://nodejs.org/en/download/) and ensure you are able to run the `npm` command in a command prompt:
-
-   ```bash
-   npm --version
-   ```
-
-   It is recommended to have npm 7+. To update npm run `npm install -g npm`
-
-2. Install the TypeSpec compiler and libraries:
-
-```bash
-   npm init -y
-   npm install -g @typespec/compiler
 ```
-
-If you do not wish to install the compiler globally with `-g` flag, you will need to install it locally once in every TypeSpec project folder. You would also need to prefix every TypeSpec run command with `npx`. See [npx documentation](https://docs.npmjs.com/cli/v7/commands/npx)
-
-```bash
-    npx tsp init
-    npx tsp compile
+npm install -g @typespec/compiler
 ```
 
-3. Install the TypeSpec extension for your editor of choice:
-
-   - [Instructions for Visual Studio](#installing-visual-studio-extension)
-   - [Instructions for Visual Studio Code](#installing-vs-code-extension)
-
-### Creating TypeSpec project
-
-1. Create a folder for your new TypeSpec project
-
-2. Initialize a TypeSpec project.
-
-   - Run `tsp init` > Select `Generic REST API` template with `@typespec/rest` and `@typespec/openapi3` libraries checked.
-   - Run `tsp install` to install Node package dependencies.
-
-3. Open the folder in your editor and edit `main.tsp`
-
-4. Follow our [documentation](https://microsoft.github.io/typespec) to get started writing TypeSpec!
-
-5. Once you're ready to compile your TypeSpec to Swagger, save the file and type this at the command prompt in your project folder:
-
-   ```bash
-   tsp compile .
-   ```
-
-   This will compile the TypeSpec files in the project folder into one output file: `.\tsp-output\openapi.json`.
-
-## Troubleshooting
+#### Tools
 
-[See common issues here](./troubleshooting.md)
-
-## Usage
-
-See full usage documentation by typing:
+[VSCode](https://typespec.io/docs/introduction/editor/vscode) 
```

Comment:
We should probably take the opportunity to document that our extension is now in the marketplace and link to it /cc @timotheeguerin ?